Source/Core/DolphinQt/GameList/GameList.cpp

```diff
@@ -628,6 +637,54 @@ void GameList::OpenGCSaveFolder()
     ModalMessageBox::information(this, tr("Information"), tr("No save data found."));
 }
 
+#ifdef _WIN32
+void GameList::AddShortcutToDesktop()
```

Comment:
Consider returning a boolean and showing the message box in the caller instead to deduplicate the error message code.

The addAction call above would look like this:

```cpp
menu->addAction(tr("Add Shortcut to Desktop"), this, [this] {
  if (AddShortcutToDesktop())
  {
    ModalMessageBox::information(...);
  }
  else
  {
    ModalMessageBox::critical(...);
  }
});
```

Note that the error message box should use `critical` instead of `information` since it's an error.